prow/e2e-kind-suite.sh

```diff
@@ -60,17 +60,27 @@ done
 # shellcheck source=prow/lib.sh
 source "${ROOT}/prow/lib.sh"
 setup_kind_cluster
+setup_and_export_git_sha
 
-export HUB=${HUB:-"gcr.io/istio-testing"}
+echo 'Build'
+(cd "${ROOT}"; make build)
+
+if [[ -n $(git diff) ]]; then
```

Comment:
Why do we need this check?

Reply:
oops it is not needed.. That check is needed for presubmit testing(copy pasted it from istio-presubmit.sh). I have deleted since we are planning to add this test ONLY in POSTsubmit suite.